util/docker/docker_test.go

```diff
@@ -0,0 +1,156 @@
+package docker
+
+import (
+	"os"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/DataDog/datadog-process-agent/util"
+	"github.com/docker/docker/api/types"
+	dockernetwork "github.com/docker/docker/api/types/network"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFindDockerNetworks(t *testing.T) {
+	assert := assert.New(t)
+
+	hostProc := "/tmp/test-find-docker-networks/proc/"
```

Comment:
Yes this is definitely true. As far as I can tell, tests are not run in parallel by default so I think we're ok for now. Let me know if you've found otherwise on parallel tests.